build.proj

```diff
@@ -33,8 +33,10 @@
     <ExeExtension>.exe</ExeExtension>
     <ExeExtension Condition=" '$(OS)' != 'Windows_NT' "></ExeExtension>
     <DotnetStage0>$(Stage0Path)/dotnet$(ExeExtension)</DotnetStage0>
-
     <DotnetCliBuildDirectory>$(RepoRoot)/build_projects/dotnet-cli-build</DotnetCliBuildDirectory>
+    <OutputDirectory>$(RepoRoot)/artifacts</OutputDirectory>
+    <PackagesDir>$(RepoRoot)/.nuget</PackagesDir>
```

Comment:
Can we place this in some `BuildTools.props` file then so we can get context from the filename if not the property itself.

---
In reply to: [72140951](https://github.com/dotnet/cli/pull/3925#discussion_r72140951) [](ancestors = 72140951)